src/server/middleware/symbolicateMiddleware.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ */
+
+const SourceMapConsumer = require('source-map').SourceMapConsumer;
+const path = require('path');
+const delve = require('dlv');
+const messages = require('../../messages');
+const logger = require('../../logger');
+
+/**
+ * Create an Express middleware for handling React Native symbolication requests
+ *
+ * @param {*} compiler - WebPack compiler.
+ * @return {function} The Express middleware.
```

Comment:
We use `Flow` in other files, so I am thinking instead of type annotations we could use Flow instead?